apps/yapms/src/lib/components/modals/editregionmodal/EditRegionModal.svelte

```diff
@@ -34,12 +35,18 @@
 			type="number"
 			placeholder="Value"
 			class="input input-bordered w-full"
-			min="1"
+			min="0"
+			required
+			on:input={(event) => {
+				$EditRegionModalStore.confirmDisabled = !event.currentTarget.checkValidity();
+			}}
 			bind:value={newValue}
 		/>
 		<div class="modal-action">
 			<button class="btn btn-primary" on:click={close}> No </button>
-			<button class="btn btn-success" on:click={confirm}> Update </button>
+			<button class="btn btn-success" on:click={confirm} disabled={confirmDisabled}>
```

Comment:
I think what you could do here is simply check if the newValue is out of bounds, set disabled to true.